extractor/src/main.py

```diff
@@ -187,10 +187,20 @@ def broadcast_events(dataframe, analysis, batch_id=None):
                 event["resource_type"] = resource_type
                 event["resource_id"] = resource_id
                 event["record"] = record
-                producer.produce_event(
-                    topic=f"{PRODUCED_TOPIC_PREFIX}{batch_id}",
-                    event=event
-                )
+                try:
```

Comment:
is it worth adding a try block here since there is already one in the outer scope ? maybe we could catch just after EmptyResult.